core/client/fs/src/test/java/alluxio/client/block/policy/CapacityBaseRandomPolicyTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.block.policy;
+
+import alluxio.client.block.BlockWorkerInfo;
+import alluxio.client.block.policy.options.GetWorkerOptions;
+import alluxio.wire.WorkerNetAddress;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class CapacityBaseRandomPolicyTest {
+
+  @Test
+  public void getSuccessWorker() {
+    GetWorkerOptions getWorkerOptions = GetWorkerOptions.defaults();
+    ArrayList<BlockWorkerInfo> blockWorkerInfos = new ArrayList<>();
+    WorkerNetAddress netAddress1 = new WorkerNetAddress();
+    WorkerNetAddress netAddress2 = new WorkerNetAddress();
+    WorkerNetAddress netAddress3 = new WorkerNetAddress();
+    WorkerNetAddress netAddress4 = new WorkerNetAddress();
```

Comment:
I suspect all these empty `WorkerNetAddress` equal to each other. Could you double check?

Reply:
thanks for your good suggestions. I done with assertSame instead.